app/jobs/vendor_proof_job.rb

```diff
@@ -1,9 +1,8 @@
 class VendorProofJob
-  def self.perform(document_capture_session_id, stages)
+  def self.perform_resolution_proof(document_capture_session_id, proof_state_id)
```

Comment:
`proof_state_id` sounds like an ID attribute to me, but this is a boolean, can we make it something like `should_proof_state_id`